client/components/sidebar.jsx

```diff
@@ -111,6 +132,8 @@ export const Sidebar = () => {
           <li>{accessTypes}</li>
           {assignments ? <li>Assignments</li> : <div></div>}
           {certificateOfCompletion ? <li>Certificate of Completion</li> : <div></div>}
+          {totalArticles > 0 ? <li>{totalArticles} articles</li> : <div></div>}
+          {courseLength}
```

Comment:
do you need an empty <div> each time? null could also be an option. I guess ternary operators are never the most elegant solution when we only really want one outcome. Another option (if you only want something rendered if there is data):

`{totalArticles > 0 && <li>{totalArticles} articles</li>}`

...only renders `<li>{totalArticles} articles</li>` if `totalArticles > 0` (Kyle taught me that!)